src/HonzaBotner.Discord.Services/EventHandlers/PollReactionsHandler.cs

```diff
@@ -8,27 +8,24 @@ namespace HonzaBotner.Discord.Services.EventHandlers;
 
 public class PollReactionsHandler : IEventHandler<MessageReactionAddEventArgs>
 {
-    public async Task<EventHandlerResult> Handle(MessageReactionAddEventArgs args)
+    public Task<EventHandlerResult> Handle(MessageReactionAddEventArgs args)
     {
-        if (args.User.IsBot) return EventHandlerResult.Continue;
+        if (args.User.IsBot) return Task.FromResult(EventHandlerResult.Continue);
         _ = Task.Run(() => HandleAsync(args));
-        await Task.Delay(0);
-        return EventHandlerResult.Continue;
+        return Task.FromResult(EventHandlerResult.Continue);
     }
 
     private async Task HandleAsync(MessageReactionAddEventArgs args)
     {
-        DiscordMessage message = args.Message.Content is null
-            ? await args.Channel.GetMessageAsync(args.Message.Id)
-            : args.Message;
+        DiscordMessage message = await args.Channel.GetMessageAsync(args.Message.Id);
```

Comment:
Could this throw?

Reply:
```csharp
/// <exception cref="T:DSharpPlus.Exceptions.UnauthorizedException">Thrown when the client does not have the <see cref="F:DSharpPlus.Permissions.ReadMessageHistory" /> permission.</exception>
/// <exception cref="T:DSharpPlus.Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
/// <exception cref="T:DSharpPlus.Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
/// <exception cref="T:DSharpPlus.Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
```
It could

Reply:
Done